src/main/java/org/folio/service/orders/flows/update/open/OpenCompositeOrderFlowValidator.java

```diff
@@ -173,43 +179,70 @@ private Future<Void> validateLocationRestrictions(CompositePoLine poLine, List<F
    * The method checking fund location against valid locations and holding locations in POL to identify restricted locations
    * <br> if there is one, will be stored to put in error as parameter.
    * <br> otherwise, order can be opened
-   * @param poLine poLine of order that requested to be open
+   *
+   * @param poLine          poLine of order that requested to be open
    * @param restrictedFunds restricted funds of poLine
-   * @param requestContext requestContext
+   * @param requestContext  requestContext
    * @return Restricted locations
```

Comment:
if this docs is not helpful, when you first to read the code, you can remove. I hope code itself explain to reader its logic

Reply:
Doc looks good to me as a short description